### PR TITLE
Kills the last remains of Christmas

### PR DESCRIPTION
**Pull request by taukausanake**
Fixes: #8702
Fixes: #8564
Fixes: #8557

I _really_ hope I didn't fuck anything up

:cl: 
Fix: Removes remaining snow on the Cyberiad and its minimap
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Also, someone remind me what `warning: LF will be replaced by CRLF in _maps/map_files/cyberiad/cyberiad.dmm` means. I know it's not liked much but forget why. Something to do with newlines?

**Comment by MarcellusPye** [comment]
Changelog plz

**Comment by taukausanake** [comment]
@MarcellusPye I thought fixes didn't need changelogs? Don't troll me here I'm not smart

**Comment by tigercat2000** [comment]
@taukausanake We prefer basically everything has changelogs, the only exception is critical exploit fixes or refactors that affect nothing

**Comment by taukausanake** [comment]
Done and done

**Comment by tigercat2000** [comment]
@taukausanake Also, the LF/CRLF thing is just git trying to convert your file endings to the correct ones for the repo - which are barely standardized. It's safe to ignore.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -1969,7 +1969,7 @@
...
 "aLU" = (/obj/machinery/light{dir = 8},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/atmospherics/pipe/simple/hidden/supply,/turf/simulated/floor/plasteel{dir = 8; icon_state = "neutral"},/area/crew_quarters/fitness)
-"aLV" = (/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/door/firedoor,/obj/machinery/door/airlock/public/glass{name = "Mr. Chang's"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/structure/disposalpipe/segment,/turf/simulated/floor/snow,/area/crew_quarters/mrchangs)
+"aLV" = (/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/door/firedoor,/obj/machinery/door/airlock/public/glass{name = "Mr. Chang's"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/structure/disposalpipe/segment,/turf/simulated/floor/wood,/area/crew_quarters/mrchangs)
```
Comment (taukausanake) [inline _maps/map_files/cyberiad/cyberiad.dmm]: Wait WHY DID I MAP MERGE THIS IF THIS WAS ALL I NEEDED TO DO

**Comment by SamHPurp** [comment]
my dude, you appear to have ran the map generator before fixing the snow.... the mr changs snow is still on your new map

**Comment by taukausanake** [comment]
I'm pretty sure I did the thing I was supposed to do. I ran the thing first, made the edits, saved, and ran again. The file edited after all, shouldn't it have updated the map too?

**Comment by Fox-McCloud** [comment]
addressed in another PR.

**Pull request closed without merging**